lib/exit.js

```diff
@@ -46,4 +47,29 @@ export default class Exit {
     }
     return output;
   }
+
+  static txFromProof(proof) {
+    const tx = Tx.fromRaw(this.parseTxDataFromProof(proof));
```

Comment:
`Tx.parseTxDataFromProof` (it's a static method, no this)

Reply:
@sunify otherwise i get
```
    ReferenceError: parseTxDataFromProof is not defined
      at Function.txFromProof (lib/exit.js:66:15)
      at Context.<anonymous> (lib/exit.spec.js:90:21)
```